config/edge-api/server.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: edge-api
+  namespace: system
+  labels:
+    app: flotta-edge-api
+spec:
+  selector:
+    matchLabels:
+      app: flotta-edge-api
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: flotta-edge-api
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      containers:
+        - command:
+            - /flotta-api
+          envFrom:
+            - configMapRef:
+                name: manager-config
+          image: edge-api:latest
+          imagePullPolicy: IfNotPresent
+          name: http
+          ports:
+            - containerPort: 8043
+              name: yggds
+            - containerPort: 8080
+              name: metrics
+          securityContext:
+            allowPrivilegeEscalation: false
+          resources:
+            limits:
+              cpu: 500m
+              memory: 1Gi
+            requests:
+              cpu: 100m
+              memory: 256Mi
+      serviceAccountName: controller-manager
```

Comment:
for the flotta-edge-api we need more restricted set of roles and a different service account.

Reply:
Will do! Thanks for reminding me!:)